scrapers/upskirtjerk.yml

```diff
@@ -0,0 +1,29 @@
+name: Upskirt Jerk
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - upskirtjerk.com/videoentry/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+     Title:
+        selector: //h1[@class='page_title']/text()
+     Date:
+       selector: //span[@class="day"]/span/text()|//span[@class="day"]/text()
+       concat: "/"
+       postProcess:
+         - replace:
+             - regex: ^(\w+)/(\d+)$ # for dates with current year
+               with: $1/21/$2 # the year part needs to be updated every year
+         - parseDate: Jan/06/02
+     Performers:
+      Name: //span[@class="meta_modelcategory meta_category"]//a
+     Details: //meta[@property="og:description"]/@content
```

Comment:
The meta descriptions are partial, not complete
```yaml
     Details: //div[@class="post_excerpt"]/p
```
seems to work, we need to check though if some scenes have multiple `<p>` and need a concat

Reply:
I've checked a number of scenes, including newer and older scenes. It looks like they always have everything on a single `<p>` for video descriptions